### PR TITLE
Create te.ps4

### PR DESCRIPTION
**Pull request by VihaanReddyM**
added telugu as a lanuage

### CODE REVIEW

**Comment by amd64fox** [comment]
Thanks

Most likely, you used an outdated template for translation. [Here's](https://github.com/amd64fox/SpotX/blob/main/scripts/installer-lang/en.ps1) the latest English version. Could you update your translation based on the latest en.ps1 file?

Also, are you sure the short code for Telugu language is "tl" and not "te"?

**Comment by VihaanReddyM** [comment]
ok

**Comment by VihaanReddyM** [comment]
I can do more indian languages if you want sir

**Comment by amd64fox** [comment]
Your translation is still not up-to-date, please perform line-by-line comparison with the en.ps1 file linked above.

Also, the file extension is incorrect.

File: scripts/installer-lang/te.ps1 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    PodcatsSelect   = "మీరు పాడ్‌కాస్ట్‌లు, ఎపిసోడ్‌లు మరియు ఆడియోబుక్‌లను ప్రధాన పేజీనుండి వేయాలా? [Y/N]"
+    DowngradeNote   = "స్పాటిఫై డౌన్‌గ్రేడ్ అయ్యిందానికి సూచించినాము"
+    UpdBlock        = "స్పాటిఫై నవీకరణలను నిలిపివేయాలి"
+    UpdUnblock      = "స్పాటిఫై నవీకరణలను నిలిపివేయకపోయింది"
+    UpdSelect       = "మీరు స్పాటిఫై నవీకరణలను నిలిపివేయాలా? [Y/N]"
+    ModSpoti        = "స్పాటిఫైని ప్యాచింగ్ చేస్తోంది..."
+    Error           = "లోపం"
+    FileLocBroken   = "స్పాటిఫై ఫైళ్‌ల స్థానం పారావారం చేసారు, దయచేసి స్పాటిఫై క్లయింట్‌ను అనిస్తాల్ చేసి Install.bat ను మళ్లీ రన్ చేయండి"
+    Spicetify       = "స్పైస్‌టిఫై కనిపించింది, అదనపుతున్నది స్పాటిఫై తరువాత ఇన్‌ఫోలును FAQలో ఉంచుకోవాలా? [Y/N]"
+    NoRestore       = "స్పాటిఫై క్లయింట్‌ను అనిస్తాల్ చేయాలనుకుంటున్నాము, xpui.bak కనుగొనలేదు. `nదయచేసి Install.bat ను అనిస్తాల్ చేయండి"
```
Comment (amd64fox) [inline scripts/installer-lang/te.ps1:43]: wrong translation

File: scripts/installer-lang/te.ps1 [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    PodcatsOff      = "పాడ్‌కాస్ట్‌లు ఆఫ్‌లో ఉన్నాయి"
+    PodcastsOn      = "పాడ్‌కాస్ట్‌లు ఆన్‌లో ఉన్నాయి"
+    PodcatsSelect   = "మీరు పాడ్‌కాస్ట్‌లు, ఎపిసోడ్‌లు మరియు ఆడియోబుక్‌లను ప్రధాన పేజీనుండి వేయాలా? [Y/N]"
+    DowngradeNote   = "స్పాటిఫై డౌన్‌గ్రేడ్ అయ్యిందానికి సూచించినాము"
+    UpdBlock        = "స్పాటిఫై నవీకరణలను నిలిపివేయాలి"
+    UpdUnblock      = "స్పాటిఫై నవీకరణలను నిలిపివేయకపోయింది"
+    UpdSelect       = "మీరు స్పాటిఫై నవీకరణలను నిలిపివేయాలా? [Y/N]"
+    ModSpoti        = "స్పాటిఫైని ప్యాచింగ్ చేస్తోంది..."
+    Error           = "లోపం"
+    FileLocBroken   = "స్పాటిఫై ఫైళ్‌ల స్థానం పారావారం చేసారు, దయచేసి స్పాటిఫై క్లయింట్‌ను అనిస్తాల్ చేసి Install.bat ను మళ్లీ రన్ చేయండి"
```
Comment (amd64fox) [inline scripts/installer-lang/te.ps1:41]: wrong translation

**Comment by amd64fox** [comment]
Oh, I started to check the correctness of the translation and found that there are serious errors in almost every translated line that change the meaning of the text. For now, I will close this. As soon as you provide a more accurate translation, please create a new pr. 

Thanks

**Comment by VihaanReddyM** [comment]
ok sorry for the mistakes

**Pull request closed without merging**